package.json

```diff
@@ -13,10 +13,11 @@
     "extends": "react-app"
   },
   "devDependencies": {
+    "react-localize": "github:eggheadio/react-localize#2.0.2",
     "react-scripts": "^0.8.4"
   },
   "dependencies": {
-    "egghead-ui": "^4.3.0",
+    "egghead-ui": "^4.3.1",
```

Comment:
Upgraded egghead-ui to have better console errors for `styled-components` and to allow `<Text />` from `react-localize` in place of plain strings.